sources/web/datalab/polymer/components/resizable-divider/resizable-divider.ts

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Resizable divider element for Datalab.
+ * This element is a container for two elements with a vertical resize divider.
+ */
+class ResizableDividerElement extends Polymer.Element {
+
+  /**
+   * Minimum pane width in pixels. Defaults to 50(px);
+   */
+  public minimumWidthPx: number;
+
+  /**
+   * Initial position of the divider in percentage. Defaults to 50(%);
+   */
+  public initDividerPosition: number;
+
+  private _boundMouseDownHandler: EventListenerOrEventListenerObject;
+  private _boundMouseupHandler: EventListenerOrEventListenerObject;
+  private _boundMouseMoveHandler: EventListenerOrEventListenerObject;
+  static get is() { return 'resizable-divider'; }
+
+  static get properties() {
+    return {
+      initDividerPosition: {
+        type: Number,
+        value: 50,
+      },
+      minimumWidthPx: {
```

Comment:
Would it be a lot more work to make this just minimumWidth and have it accept either "100px" or "50%" as a value? That would make it more consistent with the other width specifications in CSS.